Dockerfile.concourse

```
@@ -0,0 +1,8 @@
+# The value of ECR_REPOSITORY_URI will be substituted by the Concourse build
+FROM {{ECR_REPOSITORY_URI}}/onsdigital/dp-concourse-tools-ubuntu-22:ubuntu22.4-jammy-20250126
+
+WORKDIR /app/
+
+COPY dis-redirect-api .
+
+CMD ["./dis-redirect-api"]
```

To match redirect proxy
```suggestion
CMD ./dis-redirect-api
```